jobs/jobs.go

```diff
@@ -1,26 +1,41 @@
 package jobs
 
 import (
+	"time"
+
 	"github.com/flanksource/commons/logger"
+	"github.com/patrickmn/go-cache"
 	"github.com/robfig/cron/v3"
 )
 
 const (
-	TeamComponentOwnershipSchedule = "@every 15m"
+	TeamComponentOwnershipSchedule  = "@every 15m"
+	EvaluateEvidenceScriptsSchedule = "@every 5m"
 )
 
+// Cache for storing compiled CEL scripts
+var prgCache *cache.Cache
+
 var FuncScheduler = cron.New()
 
-func ScheduleFunc(schedule string, fn func()) (interface{}, error) {
+func ScheduleFunc(schedule string, fn func()) (any, error) {
 	return FuncScheduler.AddFunc(schedule, fn)
 }
 
 func Start() {
-	// running first at startup and then with the schedule
+	prgCache = cache.New(24*time.Hour, 1*time.Hour)
```

Comment:
I don't think this should be in this file, rather in the file where the cache is used. 
Alternatively in a cel/cache.go which is used by a cel.Evaluate(script, args) which becomes reusable for other use cases.

Reply:
For now, I'll just put this in the same file. I originally went with the `cel` package approach then realised it is too specific to evidence's scope now, so will do a refactor when we work on similar functionality again